src/main/java/com/thehrdivision/service/impl/EmployeeRemoveServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.thehrdivision.service.impl;
+
+import com.thehrdivision.repository.EmployeeRepository;
+import com.thehrdivision.service.EmployeeRemoveService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class EmployeeRemoveServiceImpl implements EmployeeRemoveService {
+
+    private final EmployeeRepository repository;
+
+    @Override
+    public boolean successfull(Integer id) {
+        if (repository.existsById(id)) {
+            repository.deleteById(id);
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean successfull() {
```

Comment:
Niepoprawna nazwa metodki. Cza zmianić na: `execute` z uwagi na nazwę akcji w nazwie klasy. Czy programista używając metodki `successful` będzie wiedział, że ta metodka usuwa wszystkie obiekty z repo? W najlepszym wypadku pójdzie do definicji klasy i to sprawdzi co się wiąże ze **stratą czasu** lub **użyje jej źle** i w jakimś warunku przy okazji usunie wszystkie obiekty..

Reply:
Analizując Twoją wypowiedź, która ma wiele słusznych uwag myślę ogólnie o nazewnictwie metod zwracających boolean'y: https://stackoverflow.com/questions/3874350/naming-conventions-for-java-methods-that-return-boolean
i wynika z tego, że metoda mogłaby zostać przemianowana na ```boolean isExecuted()```, czy to jest ok, według Ciebie?